test/difficulty_index/test_difficulty_index_plotting.py

```diff
@@ -0,0 +1,15 @@
+import example_difficulty_index as edi  
+from metcalcpy.compare_images import CompareImages
+import pytest
+
+def test_difficulty_index_plot():
+    """
+    Compare the expected image (diff_index_expected.png) to the latest
+    one generated.  Invoke the plot_difficulty_index.py
+    script to generate the difficulty_index.png
+    """
+    file1 = 'swh_North_Pacific_5dy_ensemble.npz'
+    edi.main()
+    comparison = CompareImages('diff_index_expected.png', 'swh_North_Pacific_difficulty_index_9_00_feet.png')
+    assert comparison.mssim == 1
+
```

Comment:
There is an assertion error I encounter with this test:
>       assert comparison.mssim == 1
E       assert 0.8429471911042749 == 1
E        +  where 0.8429471911042749 = <metcalcpy.compare_images.CompareImages object at 0x11ba20860>.mssim